package.json

```diff
@@ -36,5 +40,8 @@
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
     "webpack": "^3.11.0"
+  },
+  "jest": {
+    "setupTestFrameworkScriptFile": "<rootDir>test/setupTests.js"
   }
 }
```

Comment:
You've `npm i --save`d a lot of these when they could be just `npm i --save-dev`; you can actually just do `npm i --save-dev` for the ones that you want to save to dev dependencies (if you have any dev dependencies you want to switch to actual dependencies, you can do `npm i --save-prod`)